addons/account/models/account.py

```diff
@@ -792,7 +792,7 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
             tax_base = base
 
             if tax.include_base_amount:
-                base += tax_amount
+                base += tax_amount if not tax.price_include or tax.amount_type != 'percent' else 0
```

Comment:
ok for the price_include clause, but what about the amount_type != percent?

according to me, it should be:
```python
base += tax_amount if not tax.price_include else 0
```